packages/core/src/Search.cpp

```diff
@@ -343,11 +358,11 @@ void Search::Implementation::PathSet::add_path(const std::string& path)
   if(path.empty())
     return;
 
-  if(path.at(0) != '/')
+  if(IS_NOT_ABSOLUTE_PATH(path))
   {
-    std::cerr << "[soss error] Attempting to add a prefix path that is not an "
-              << "absolute path: " << path << std::endl;
-    return;
+  std::cerr << "[soss error] Attempting to add a prefix path that is not an "
```

Comment:
It looks like this block of code was accidentally un-indented.

Reply:
Yes, sorry. Done.